java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -260,4 +265,93 @@ public void info_with_multi_node_route_and_options() {
             }
         }
     }
+
+    @Test
+    @SneakyThrows
+    public void configGet_with_no_args_returns_error() {
+        var exception =
+                assertThrows(
+                        ExecutionException.class, () -> clusterClient.configGet(new String[] {}).get());
+        assertTrue(exception.getCause() instanceof RedisException);
+        assertTrue(exception.getCause().getMessage().contains("wrong number of arguments"));
+    }
+
+    @Test
+    @SneakyThrows
+    public void configGet_with_wildcard() {
+        var data = clusterClient.configGet(new String[] {"*file"}).get();
+        assertTrue(data.size() > 5);
```

Comment:
The IT tests on python/node don't check this.  Are we sure this will be true on all systems?

Reply:
`/systems/redis versions/g`
yes
if something changes (e.g. a new version added to CI), we can update the test

Reply:
If the goal is to ensure fetching more than a single value with the wildcard, asserting that data.size > 1 suffices. Let's avoid making these tests too dependent on Redis's current configurations to minimize the need for adjustments when client-irrelevant changes occur